core/src/test/java/org/testcontainers/dockerclient/AmbiguousImagePullTest.java

```diff
@@ -16,14 +16,14 @@
     public void testNotUsingParse() {
         DockerClient client = DockerClientFactory.instance().client();
         List<Image> alpineImages = client.listImagesCmd()
-            .withImageNameFilter("alpine:latest")
+            .withImageNameFilter("testcontainers/helloworld:latest")
```

Comment:
We never use the `latest` tag of this image anywhere else, so we can safely use it here without worrying about it being recorded in the cache of available images.